### ISSUE TITLE
Custom tasks script icons?

### ISSUE THREAD

**Issue by andmish** [limbonaut/limboai#140]
I've noticed I often do right clicking on a task and picking `Edit Script` in trees to jump to the task's script quickly.
So now I wonder what are those selectable empty cells on the right? Maybe is it possible to use them to show script icon (similar to Godot's scene viewer)? And clicking on it will open that script? Just thought it could be a handy "2 clicks to 1 click" shortcut, also quick visualizer which tasks are yours 

![icon_place](https://github.com/limbonaut/limboai/assets/37307893/402711d5-905b-40f4-8773-1a6810ff5978)

![icon_example](https://github.com/limbonaut/limboai/assets/37307893/de009b4e-b928-443b-8375-9b1433007826)

**Comment by limbonaut** [comment]
Usefulness depends on your aspect ratio. It's going to be way too far on the right, if you have 21:9 and wider. But I guess it's not such a common display type.

**Comment by andmish** [comment]
@limbonaut yeah, it could be much better to add narrow column at the left and show script icons there, but I don't know how this tree view is working, will it be possible to sync icon visibility (collapsing rows in that column) with folding/unfolding the tree?
 
![icon_example_left](https://github.com/limbonaut/limboai/assets/37307893/7783f18e-6c5f-4e10-bb80-c8984ab48571)

but if we will look at godot's text editor as tree view (and breakpoints as like task script icons), breakpoints are folding/unfolding nicely, so maybe it could be possible? 🤔
![tree_view](https://github.com/limbonaut/limboai/assets/37307893/4b70c5e3-2167-432c-9ff2-9c8b8db6ed93)

**Comment by limbonaut** [comment]
The icon button idea is probably not the best for LimboAI editor. While I like that it would be similar to scene tree dock (familiar concept to most godoteers), but we are not in the same restricted space as the dock is. It would be awkward all the way to the right on wide and generally large displays:
![image](https://github.com/limbonaut/limboai/assets/2766569/1c1954c9-3e67-4355-bacd-689c8acfa99d)
And on the left it won't work due to how `Tree` control works. There is a possibility to implement similar technique as I used with percentages and `ProbabilitySelector`. That would require some effort, though. Maybe some of that code for percentages "widget" could be reused/refactored for this. And I'll keep looking for an alternative script indicator and quick-jump solution.
